src/components/app-top-bar.js

```diff
@@ -121,31 +121,30 @@ const CustomSuffixRenderer = ({ props, element }) => {
         element.type === EQUIPMENT_TYPES.VOLTAGE_LEVEL.type
     )
         return (
-            network && (
-                <>
-                    {element.type === EQUIPMENT_TYPES.VOLTAGE_LEVEL.type && (
-                        <IconButton
-                            disabled={
-                                networkAreaDiagramNbVoltageLevels >
-                                    NETWORK_AREA_DIAGRAM_NB_MAX_VOLTAGE_LEVELS &&
-                                networkAreaDiagramDepth !== 0
-                            }
-                            onClick={(e) =>
-                                openNetworkAreaDiagramCB(e, element)
-                            }
-                            size={'small'}
-                        >
-                            <TimelineIcon fontSize={'small'} />
-                        </IconButton>
-                    )}
+            <>
+                {element.type === EQUIPMENT_TYPES.VOLTAGE_LEVEL.type && (
                     <IconButton
-                        onClick={(e) => centerOnSubstationCB(e, element)}
+                        disabled={
+                            networkAreaDiagramNbVoltageLevels >
+                                NETWORK_AREA_DIAGRAM_NB_MAX_VOLTAGE_LEVELS &&
+                            networkAreaDiagramDepth !== 0
+                        }
+                        onClick={(e) =>
+                            openNetworkAreaDiagramCB(e, element)
+                        }
                         size={'small'}
                     >
-                        <GpsFixedIcon fontSize={'small'} />
+                        <TimelineIcon fontSize={'small'} />
                     </IconButton>
-                </>
-            )
+                )}
+                <IconButton
+                    disabled={!network && element.type === EQUIPMENT_TYPES.VOLTAGE_LEVEL.type}
```

Comment:
Looking at the code, I would have written 
`element.type !== EQUIPMENT_TYPES.SUBSTATION.type`
instead of 
`element.type === EQUIPMENT_TYPES.VOLTAGE_LEVEL.type`
because you can still have a null exception here no ?

Reply:
Ah ok I suspected it, indeed it's right below my bad, but yeah maybe change the condition in centerOnSubstationCB then ?
```
                element.type === EQUIPMENT_TYPES.VOLTAGE_LEVEL.type
                    ? network.getVoltageLevel(element.id).substationId;
                    : element.id
```